client/components/command-pallette/README.md

```diff
@@ -0,0 +1,69 @@
+# Command Palette
+
+The Command Palette is a React component used to display a text input used to search and filter a list of commands. It can be accessed in the Calypso using the shortcut `cmd+k`.
+
+The command palette is inspired by [Core](https://github.com/WordPress/gutenberg/blob/trunk/packages/commands/README.md) implementation used in the Gutenberg editor. We use the same CSS and similar markup, the commands share the same basic props that we extended to support the following differences:
+
+- The Calypso version supports multisite commands with the sites as second step commands.
+- Users can go back to root commands with the `esc` and `backspace` keys, or the back button.
+- We change the search placeholder to inform the user about the current command.
+- Besides of icons, we also support images for the commands.
+- We support two lines of text for the commands.
+
```

Comment:
Can we include some mention of https://github.com/WordPress/gutenberg/issues/55514 and how we'd eventually like to reconcile the differences?